pxr/usd/bin/usddiff/usddiff.py

```diff
@@ -170,16 +172,33 @@ def listFiles(dirpath):
                     for file in files]
         return set(ret)
 
-    # Must have FILE FILE, DIR DIR, DIR FILES... or FILES... DIR.
+    origArgs = args
+
+    # path may not be a "real" path due to custom resolvers... so use
+    # Sdf.Layer.FindOrOpen first
+    args = []
+    for arg in origArgs:
+        layer = Sdf.Layer.FindOrOpen(arg)
```

Comment:
I think this would break usddiff in the case where you pass in directories to compare, since args (and origArgs) would contain directory names. You would wind up calling Sdf.Layer.FindOrOpen on a directory, which would fail and result in a ValueError getting raised.